src/lib.rs

```diff
@@ -481,6 +484,37 @@ pub fn main() -> Result<(), Error> {
             Ok(())
         }
 
+        Commands::Delete {
+            db,
+            genome_ids,
+            output_file,
+        } => {
+            let ref_db = utils::strip_sketch_extension(db);
```

Comment:
Looks like you already ignore the extension anyway? (see comment on cli help above)